vitrina/datasets/helpers.py

```diff
@@ -24,10 +24,13 @@ def generate_dataset_name(organization: Organization, dataset_title: str) -> str
         str: A string in the format "organization_part/dataset_part", where both parts are slugified.
     """
     slugify_ascii_lower = partial(slugify, lowercase=True, allow_unicode=False)
-    organization_part = organization.name or organization.slug or organization.title
-    organization_part = slugify_ascii_lower(organization_part)
     dataset_part = slugify_ascii_lower(dataset_title)
-    return f"datasets/gov/{organization_part}/{dataset_part}"
+    if organization.name:
+        return f"{organization.name}{dataset_part}"
+    organization_part = organization.slug or organization.title
+    organization_part = slugify_ascii_lower(organization_part)
+    prefix = "datasets/gov" if organization.kind == Organization.GOV else "datasets/org"
+    return f"{prefix}/{organization_part}/{dataset_part}"
```

Comment:
First suggestion to avoid repetition:

```suggestion
    organization_kind = "gov" if organization.kind == Organization.GOV else "org"
    return f"datasets/{organization_kind}/{organization_part}/{dataset_part}"
```

Another one:
- Can't we just use `organization.kind` here as a value? Organization might have new `kinds` introduced later on and this part would need changing. Keep in mind that I do not have the context for this and you might have intended to do this intentionally:D

Reply:
Organizations have 3 kinds, and only 2 prefixes, so thats why I've done it